src/browser/ui/dom/components/ReactDOMButton.js

```diff
@@ -44,7 +44,7 @@ var ReactDOMButton = ReactClass.createClass({
   mixins: [AutoFocusMixin, ReactBrowserComponentMixin],
 
   render: function() {
-    var props = {};
+    var props = filterDisabledEvents(this.props);
 
     // Copy the props; except the mouse listeners if we're disabled
     for (var key in this.props) {
```

Comment:
Correct. I missed this when I transposed over from my dead fork :-/. Good catch!